frontend/packages/noobaa-storage-plugin/src/components/status-card/statuses.tsx

```diff
@@ -1,77 +1,51 @@
 import * as _ from 'lodash';
+import { PrometheusHealthHandler } from '@console/plugin-sdk';
 import { HealthState } from '@console/shared/src/components/dashboard/status-card/states';
-import { PrometheusResponse } from '@console/internal/components/graphs';
-import { FirehoseResult } from '@console/internal/components/utils';
 import { getGaugeValue } from '../../utils';
 
-const NooBaaStatus = [
-  {
-    state: HealthState.ERROR,
-    message: 'MCG is not running',
-  },
-  {
+const NooBaaStatus = {
```

Comment:
This mapping does not make sense. Just use an array.

Reply:
I dont care whatever but having a mapping makes me understand whats coming from backend.